docs/interact/programmability/lua-debugging.md

```diff
@@ -88,55 +88,55 @@ Redis Lua debugger help:
 [b]reak 0            Remove all breakpoints.
 [t]race              Show a backtrace.
 [e]val <code>        Execute some Lua code (in a different callframe).
-[r]edis <cmd>        Execute a Redis command.
-[m]axlen [len]       Trim logged Redis replies and Lua var dumps to len.
+[r]edis <cmd>        Execute a Valkey command.
+[m]axlen [len]       Trim logged Valkey replies and Lua var dumps to len.
                      Specifying zero as <len> means unlimited.
 [a]bort              Stop the execution of the script. In sync
                      mode dataset changes will be retained.
 
 Debugger functions you can call from Lua scripts:
-redis.debug()        Produce logs in the debugger console.
-redis.breakpoint()   Stop execution as if there was a breakpoint in the
+valkey.debug()        Produce logs in the debugger console.
+valkey.breakpoint()   Stop execution as if there was a breakpoint in the
                      next line of code.
 ```
 
 Note that when you start the debugger it will start in **stepping mode**.
 It will stop at the first line of the script that actually does something before executing it.
 
 From this point you usually call `step` in order to execute the line and go to the next line.
-While you step Redis will show all the commands executed by the server like in the following example:
+While you step Valkey will show all the commands executed by the server like in the following example:
 
 ```
 * Stopped at 1, stop reason = step over
--> 1   redis.call('ping')
+-> 1   valkey.call('ping')
```

Comment:
I think this PR can be replaced by #38. (We use `server.call` in Lua, and other changes.)

Sorry @hwware, I forgot about this PR. Actually, these PRs are easy to write and hard to review. In my recent PRs, I wrote some more exact regular expressions and did more automatic search-replace.

Reply:
No problem, i think this kind of pr should be split into smaller prs.  ^-^

Reply:
What I did in some PRs: I worked on good regular expressions, tested them and reviewed the result, but I didn't commit the changes. When it was good enough, I did the change fast. I think it works.